Project/src/MakeCall/CallCard.js

```diff
@@ -643,6 +650,13 @@ export default class CallCard extends React.Component {
     }
 
     async handleClickEmoji(index) {
+        
+        if(this.callJoinType === CallJoinType.TeamsCall && !this.state.canReact) {
```

Comment:
What does the feature do when user tries to react on a 1:1 call? Do we throw expception. Thinking why client application has do the resolution and determine if reactions will fail on a 1:1 call.

Reply:
It will throw an error for 1:1 call.